modules/apps/commerce/commerce-pricing-web/src/main/resources/META-INF/resources/commerce_price_lists/commerce_price_entry/details.jspf

```diff
@@ -39,6 +41,7 @@ boolean discountDiscovery = BeanParamUtil.getBoolean(commercePriceEntry, request
 		<div class="col-4">
 			<aui:input label="<%= priceInputLabel %>" name="price" suffix="<%= HtmlUtil.escape(commerceCurrency.getCode()) %>" type="text" value="<%= commerceCurrency.round(commercePriceEntry.getPrice()) %>">
 				<aui:validator name="min">0</aui:validator>
+				<aui:validator name="max">999999999.99</aui:validator>
```

Comment:
We should create a constant containing the value 999999999.99, so that if we ever need to change it, we can do that in a single point